src/include/storage/data_table.h

```diff
@@ -213,6 +219,12 @@ class DataTable {
    */
   const BlockLayout &GetBlockLayout() const { return accessor_.GetBlockLayout(); }
 
+  /**
+   * Dump a table to disk in arrow IPC format.
```

Comment:
Can you write a high level diagram of how tables are serialized. Similar to how [`ProjectedRows`](https://github.com/cmu-db/terrier/blob/18c62295b42cc6acd06926989318bd2d94a1e416/src/include/storage/projected_row.h#L19-L23) is shown here